zim/gui/pageview/__init__.py

```diff
@@ -4516,14 +4528,14 @@ def multi_line_indent(start, end):
 			elif keyval in KEYVALS_ASTERISK + (KEYVAL_POUND,):
 				def toggle_bullet(line, newbullet):
 					bullet = buffer.get_bullet(line)
-					if not bullet and not buffer.get_line_is_empty(line):
```

Comment:
Previously, `toggle_bullet` handled checking for empty lines, resulting in different behavior between `apply_format_bullet_list`/using the Format menu and pressing `*` on a selection